{{cookiecutter.project_slug}}/ansible/deploy.yml

```diff
@@ -7,49 +7,83 @@
 - name: deploy new version of the project to the server
   hosts: web
   remote_user: buildout
+
   tasks:
 
-    - name: checkout correct version from github
+    - name: Checkout correct version from github.
       git:
         accept_hostkey: yes
-        dest: /srv/{% raw %}{{ site_name }}{% endraw %}
-        repo: ssh://git@github.com/nens/{{ cookiecutter.project_slug }}.git
-        version: "{% raw %}{{ checkout_name }}{% endraw %}"
+        dest: {{ project_path }}
+        repo: ssh://git@github.com/nens/{{ project_name }}.git
+        version: "{{ checkout_name }}"
 
-    - name: symlink buildout config
+    - name: Create var dirs if not available.
       file:
-        src: /srv/{% raw %}{{ site_name }}/{{ buildout_cfg }}{% endraw %}
-        dest: /srv/{% raw %}{{ site_name }}{% endraw %}/buildout.cfg
-        state: link
+        path: {{ project_path }}/{{ item }}
+        owner: buildout
+        group: buildout
+        mode: 0755
+        state: directory
+      with_items:
+        - etc
+        - var
+        - var/static
+        - var/log
+        - var/media
 
-    - name: bootstrap buildout (when needed)
-      shell: python3 bootstrap.py
+    - name: "Deploy (look at var/log/pipenv.log if something goes fishy)"
+      shell: pipenv install --deploy --verbose > var/log/pipenv.log 2>&1
+      environment:
+        PIPENV_VENV_IN_PROJECT: 1  # pipenv install --local is coming and will replace this
       args:
-        chdir: /srv/{% raw %}{{ site_name }}{% endraw %}
-        creates: /srv/{% raw %}{{ site_name }}{% endraw %}/bin/buildout
+        chdir: {{ project_path }}
 
-    - name: run buildout (can take a while; look at buildout.log if something goes fishy)
-      shell: bin/buildout -vv > buildout.log 2>&1
+    - name: Clean unused packages.
+      shell: pipenv clean
       args:
-        chdir: /srv/{% raw %}{{ site_name }}{% endraw %}
+        chdir: {{ project_path }}
+
+    - name: Recreate config files from templates.
+      template:
+        src: "{{ item.src }}"
+        dest: "{{ item.dest }}"
+        owner: buildout
+        group: buildout
+        mode: 0755
+      loop:
+        - { src: "templates/env.j2", dest: "{{ project_path }}/.env" }
+        - { src: "templates/supervisord.conf.j2", dest: "{{ project_path }}/etc/{{ site_name }}.supervisord.conf" }
+        - { src: "templates/nginx.conf.j2", dest: "{{ project_path }}/etc/{{ site_name }}.nginx.conf" }
 
-    - name: Run migrate
-      action: shell bin/django migrate --noinput --fake-initial
+    - name: Run migrations.
+      shell: pipenv run python manage.py migrate
```

Comment:
Why did you remove ` --noinput --fake-initial`? That used  to be necessary for the first deploy.

Reply:
--fake-initial is only necessary for existing projects, right? I thought the cookiecutter is meant for new projects, in which case you should not use the fake-initial.

Reply:
Ah, that was originally for updating older projects. Create a new project, copy/paste the ansible scripts.

Necessary when moving from south to the new django migrations. I guess we can safely remove it now :-)